apps/judicial-system/backend/src/app/modules/notification/services/caseNotification/caseNotification.service.ts

```diff
@@ -295,14 +295,14 @@ export class CaseNotificationService extends BaseNotificationService {
       theCase.courtCaseNumber ?? theCase.policeCaseNumbers[0],
     )
 
-    return this.sendEmail(
+    return this.sendEmail({
       subject,
-      body,
-      theCase.defenderName,
-      theCase.defenderEmail,
-      undefined,
-      !theCase.defenderNationalId,
-    )
+      html: body,
+      recipientName: theCase.defenderName,
+      recipientEmail: theCase.defenderEmail,
+      attachments: undefined,
```

Comment:
Consider removing undefined properties as they are no longer needed when passing in an object as a parameter.